### ISSUE TITLE
GPU utilization problrm

### ISSUE THREAD

**Issue by jwnirvana** [wutianyiRosun/CGNet#20]
Hello. The num_worker is 1 to default, then the GPU utilization is  around 25%. When the num_worker is 8,  that is around 60%. However, a new problem occurs:
=====> epoch[0/300] iter: (369/371) 	cur_lr: 0.000997 loss: 1.175 time:0.23
=====> epoch[0/300] iter: (370/371) 	cur_lr: 0.000997 loss: 0.740 time:0.23
cityscapes_train.py:42: UserWarning: volatile was removed and now has no effect. Use `with torch.no_grad():` instead.
  input_var = Variable(input, volatile=True).cuda()
[0/500]  time: 0.80
[1/500]  time: 0.02
[2/500]  time: 0.02
[3/500]  time: 0.02
[4/500]  time: 0.02
[5/500]  time: 0.02
[6/500]  time: 0.02
[7/500]  time: 0.02
[8/500]  time: 0.02
[9/500]  time: 0.02
[10/500]  time: 0.02
[11/500]  time: 0.02
[12/500]  time: 0.02
[13/500]  time: 0.02
[14/500]  time: 0.02
[15/500]  time: 0.02
[16/500]  time: 0.02
[17/500]  time: 0.02
[18/500]  time: 0.02
[19/500]  time: 0.02
[20/500]  time: 0.02
[21/500]  time: 0.02
[22/500]  time: 0.02
[23/500]  time: 0.02
[24/500]  time: 0.02
[25/500]  time: 0.02
[26/500]  time: 0.02
[27/500]  time: 0.02
[28/500]  time: 0.02
[29/500]  time: 0.02
[30/500]  time: 0.02
[31/500]  time: 0.02
[32/500]  time: 0.02
[33/500]  time: 0.02
[34/500]  time: 0.02
[35/500]  time: 0.02
[36/500]  time: 0.02
[37/500]  time: 0.02
[38/500]  time: 0.02
[39/500]  time: 0.02
[40/500]  time: 0.02
[41/500]  time: 0.02
[42/500]  time: 0.02
[43/500]  time: 0.02
[44/500]  time: 0.02
[45/500]  time: 0.02
[46/500]  time: 0.02
[47/500]  time: 0.02
[48/500]  time: 0.02
[49/500]  time: 0.02
[50/500]  time: 0.02
[51/500]  time: 0.02
[52/500]  time: 0.02
[53/500]  time: 0.02
[54/500]  time: 0.02
[55/500]  time: 0.02
[56/500]  time: 0.02
[57/500]  time: 0.02
[58/500]  time: 0.02
[59/500]  time: 0.02
[60/500]  time: 0.02
[61/500]  time: 0.02
[62/500]  time: 0.02
[63/500]  time: 0.02
[64/500]  time: 0.02
[65/500]  time: 0.02
[66/500]  time: 0.02
[67/500]  time: 0.02
[68/500]  time: 0.02
[69/500]  time: 0.02
[70/500]  time: 0.02
[71/500]  time: 0.02
[72/500]  time: 0.02
[73/500]  time: 0.02
[74/500]  time: 0.02
[75/500]  time: 0.02

I do not know what this means. Could you tell me that,please?
Thank you.
